src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -84,13 +87,13 @@ public static function createResourceForClass(\ReflectionClass $reflectionClass)
     /**
      * Wires the given definition.
      *
-     * @param string               $id
-     * @param Definition           $definition
-     * @param true|string|string[] $autowired
+     * @param string     $id
+     * @param Definition $definition
+     * @param string[]   $autowired
      *
      * @throws RuntimeException
      */
-    private function completeDefinition($id, Definition $definition, $autowired)
+    private function completeDefinition($id, Definition $definition, array $autowired = array('__construct'))
```

Comment:
since this is private, let's make the arg mandatory to make the calling code easier to read?